scripts/west_commands/runners/spi_burn.py

```diff
@@ -59,28 +59,44 @@ def do_run(self, command, **kwargs):
         else:
             self.logger.error(f'{command} not supported!')
 
-    def _flash(self):
+    def start_iceman(self):
```

Comment:
I'd suggest to move this changes to another PR, since they're completely independent (or looks like they're)

Reply:
Agree, fixed.